engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -86,80 +94,125 @@ class EngineJobExecutionActor(replyTo: ActorRef,
   // When PreparingJob, the FSM always has NoData
   when(PreparingJob) {
     case Event(BackendJobPreparationSucceeded(jobDescriptor, bjeaProps), NoData) =>
+      val updatedData = ResponsePendingData(jobDescriptor, bjeaProps)
+      // If this backend doesn't offer a cache hit copying actor, remove 'read' from the effective cache options.
+      // i.e. there's no way to copy a cache hit so don't bother looking for cache hits.  If this backend does
+      // offer a cache hit copying actor, squirrel away its props so we can create this actor later if there's a hit.
+      if (effectiveCallCachingMode.readFromCache) {
```

Comment:
Optional ToL: I'd bump this out to a helper function so that the flow of the `when` receiver is easier to follow.